src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/Color_style/Color_control.js

```diff
@@ -0,0 +1,347 @@
+define(["base/js/namespace", "jquery", "./Spectrum"], function(
+  Jupyter,
+  $,
+  Spectrum
+) {
+  "use strict";
+
+  //=============
+  $("head").append('<style type="text/css"></style>');
+  var newStyleElement = $("head").children(":last");
+  newStyleElement.html(
+    ".sp-palette .sp-thumb-el.sp-thumb-focus {outline: 1px dotted #212121;outline: 5px auto -webkit-focus-ring-color;}"
+  );
+
+  var link2 = document.createElement("link");
```

Comment:
Can we move this to use Tao's function for importing css in main.js

Reply:
done